mkt/langpacks/models.py

```diff
@@ -81,21 +84,55 @@ def handle_file_operations(self, upload):
         """Handle file operations on an instance by using the FileUpload object
         passed to set filename, size, hash on the LangPack instance, and moving
         the temporary file to its final destination."""
-        # FIXME: store validation result ?
         upload.path = smart_path(nfd_str(upload.path))
         if not self.uuid:
             self.reset_uuid()
         self.filename = self.generate_filename()
```

Comment:
I couldn't find anything else calling it and it's only a one-liner so it seems like it could just be inlined here. But if there's future plans or other things calling it I didn't find, no biggie.